src/main/java/co/novu/api/workflowoverrides/pojos/WorkflowOverride.java

```diff
@@ -0,0 +1,35 @@
+package co.novu.api.workflowoverrides.pojos;
+
+import co.novu.api.common.PreferenceSettings;
+
+import com.google.gson.annotations.SerializedName;
+
+import lombok.Data;
+
+@Data
+public class WorkflowOverride {
+
+    @SerializedName("_id")
+    private String id;
+
+    @SerializedName("_organizationId")
+    private String organizationId;
+
+    @SerializedName("_environmentId")
+    private String environmentId;
+
+    @SerializedName("_workflowId")
+    private String workflowId;
+
+    @SerializedName("_tenantId")
+    private String tenantId;
+
+    private boolean active;
+    private PreferenceSettings preferenceSettings;
+    private boolean deleted;
```

Comment:
Please use Boolean instead of primitive boolean. Do the same for other request and response classes

Reply:
Okay. Ack and modiefied.